sqlitecluster/SQLiteNode.h

```diff
@@ -112,11 +114,17 @@ class SQLiteNode : public STCPNode {
     void _onDisconnect(Peer* peer);
     void _onMESSAGE(Peer* peer, const SData& message);
 
+    SQLitePool& _dbPool;
```

Comment:
Comment please, explaining where the actual object this references is instantiated